src/main/java/org/testng/TestNGAntTask.java

```diff
@@ -1027,13 +1030,13 @@ private void readAndPrintFile(String fileName) {
       }
     }
     catch(IOException ex) {
-      ex.printStackTrace();
+      LOGGER.error(ex.getMessage(), ex);
     } finally {
       if (br != null) {
         try {
           br.close();
         } catch (IOException e) {
-          e.printStackTrace();
+          LOGGER.error(e.getMessage(), e);
```

Comment:
It is the perfect case for a `try-with-resources`.

Reply:
fixed